pilot/api/data.py

```diff
@@ -47,6 +49,51 @@ def transfer(self, files):
             raise Exception('Files dictionary does not conform: scope, name, destination')
 
 
+class StageOutClient(object):
+
+    def __init__(self, site=None):
+        super(StageOutClient, self).__init__()
+
+        # Check validity of specified site
+        self.site = os.environ.get('VO_ATLAS_AGIS_SITE', site)
```

Comment:
But do we care..? You could anyway set it like
data_client = data.StageOutClient(site='ALCF_Theta')
right?